homeassistant/components/homeassistant/strings.json

```diff
@@ -138,8 +138,41 @@
     }
   },
   "exceptions": {
+    "component_import_err": {
+      "message": "Unable to import {domain}: {error}"
+    },
+    "config_platform_import_err": {
+      "message": "Error importing config platform {domain}: {error}"
+    },
+    "config_validation_err": {
+      "message": "Invalid config for integration {domain} in file {config_file} near to line {line}: {error}. Check the logs for more information."
```

Comment:
With the reworked logging of validation errors, configuration errors should reliably be reported at the exact file and line it occurs. Please adjust the language accordingly, there's no need to say "near to line" anymore.